firebase/firestore.js

```diff
@@ -0,0 +1,43 @@
+import {auth, db} from '../firebase';
+
+export let score = 0;
+
+export const getUser = async () => {
+  try {
+    const users = db.collection('users').doc('hxTAarepPjgInAuZz5VD');
```

Comment:
Can we have this function take in the user's email or something as a parameter? Currently this is looking up the same record for every user.